ui_test/POM_hiway_asg/pages/hiway_homepage.py

```diff
@@ -0,0 +1,12 @@
+from POM_hiway_asg.pages.constant_var import timesheet_tab_btn
+import time
+
+
+class Hiway_Homepage_Options():
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class "Hiway_Homepage_Options" to match the regular expression ^[A-Z_][a-zA-Z0-9]+$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS101)